### ISSUE TITLE
Atril configuration saving

### ISSUE THREAD

**Issue by HainesB** [mate-desktop/atril#279]

**Comment by HainesB** [comment]
I inadvertently closed this issue two months ago, but the problem continues.

I go to atril View menu and set my desired configuration. Then I go to Edit, Save Current Settings as Default. These customizations are remembered if I reopen that same document. But when I open a new PDF, atril reverts to its default values. My custom settings are remembered only if I reopen a document for which I had previously saved custom settings.

I should note that I don't run any desktop environment, not Mate or any other. None of the files in /usr/bin or in /usr/share/atril appear to specify configuration values.

**Comment by shevegen** [comment]
Right. It may be useful for atril to use a fallback configuration value; could be a simple . dot file or something with a simple format. Like yaml or json or just INI style files should suffice.
